cgogn/io/mesh_io_gen.h

```diff
@@ -120,26 +130,31 @@ class MeshExport
 
 	virtual void clean_added_attributes(Map& map)
 	{
-		map.remove_attribute(indices_);
+		map.remove_attribute(vindices_);
+	}
+
+	inline std::map<Orbit, const ChunkArrayGen*> const & position_attribute() const
```

Comment:
Salut @untereiner 
Un détail (qui a son importance) : la méthode at() d'une std map lance une exception quand l'élément recherché n'en fait pas partie, contrairement à l'opérateur []. 
Je suggère de modifier cette fonction, de lui faire prendre l'orbite en paramètre, bref de faire un truc du genre : 
const ChunkArrayGen* position_attribute(Orbit orb) const
auto it = map.find(orb);
return it == map.end()?nullptr:it->second;